src/wallet/wallet2.cpp

```diff
@@ -4955,18 +4955,19 @@ bool wallet2::save_multisig_tx(const multisig_tx_set &txs, const std::string &fi
   return epee::file_io_utils::save_string_to_file(filename, ciphertext);
 }
 //----------------------------------------------------------------------------------------------------
-wallet2::multisig_tx_set wallet2::make_multisig_tx_set(const std::vector<pending_tx>& ptx_vector) const {
```

Comment:
Please revert this.

Reply:
why do you want to revert it? this function is used in `WalletImpl::createTransaction`